src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -18,12 +18,68 @@
             <h1 class="govuk-heading-m">
                 @title
             </h1>
-            <table id="pcs-details-table" class="govuk-table">
+            <table id="pcs-contact-details-table" class="govuk-table">
                 <tbody class="govuk-table__body">
-                   
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">First Name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactData.FirstName</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Last name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactData.LastName</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Position</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactData.Position</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Email</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Email</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Phone number</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Telephone</td>
+                    </tr>
                 </tbody>
             </table>
+            <h1 class="govuk-heading-m">Contact address</h1>
```

Comment:
@stevelee-civica  think for heading hierarchy this should probably be a h2, maybe worth checking with Wave

Reply:
changed to h2